cli/internal/cmd/init.go

```diff
@@ -249,6 +257,14 @@ func (i *initCmd) initialize(cmd *cobra.Command, newDialer func(validator atls.V
 		return err
 	}
 
```

Comment:
could we make the need for this more evident in the code documentation? I'd suggest a more general naming like .ApplyConfigurableNonHelmManagedResources with some doc comment. This way it's clear why we do this in the first place

Reply:
IMO `ApplyConfigurableNonHelmManagedResources` is significantly less clear than `attestationConfigApplier`.
Especially since the client currently does nothing else than applying the attestation config to the cluster.